### ISSUE TITLE
 Софт не работает ни где и ни у кого, софт никто не обновляет

### ISSUE THREAD

**Issue by mrAsh4r** [cSploit/android#1002]
DEVICE:

OS version:

**cSploit version:**

**busybox** *(installed or not):*

**Rooted with supersu?**

**logcat** *(filter it!):*
[How-to guide](https://forum.xda-developers.com/showthread.php?t=1726238)

------------------------------------------------------------------------------------------------------------------------------

**Daemon specific questions**

Go version:

OS:

Using Docker (incase of unofficial build.)

Actions performed:

Logs:
------------------------------------------------------------------------------------------------------------------------------

**Issue:**
*Remember to search for issues alike before creating new one!*

**Comment by HitItAll** [comment]
Yes, this software is no longer updated or maintained but if you bothered
to look back at all the other hundreds of issues you would know this. Also
if you had a *real* bug, then please fill out each line of the bug report.
Thanks.

**Comment by zertos1** [comment]
Софт реально не работает? Столкнулся с тем, что программа кроме роутера и моего телефона, больше ничего не видит, как быть?
